content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,143 @@ aliases:
+can be useful in A/B testing where you might want to configure traffic routes
+based on percentages across different service versions, or if you want to direct
+traffic from your internal users to a particular set of instances.
+
+With a virtual service, you can specify traffic behavior for one or more hostnames.
+You use routing rules in the virtual service that tell Envoy how to send the
+virtual service’s traffic to appropriate destinations. Route destinations can
+be versions of the same service or entirely different services.
+
+A typical use case is to send traffic to different versions of a service,
+specified as service subsets. Clients send requests to the virtual service host as if
+it was a single entity, and Envoy then routes the traffic to the different
+versions depending on the virtual service rules (for example "20% of calls go to
+the new version" or "calls from these users go to version 2"). This allows you to,
+for instance, create a canary rollout where you gradually increase the
+percentage of traffic that’s sent to a new service version. The traffic routing
+is completely separate from the instance deployment, meaning that the number of
+instances implementing the new service version can scale up and down based on
+traffic load without referring to traffic routing at all. By contrast, container
+orchestration platforms like Kubernetes only support traffic distribution based
+on instance scaling, which quickly becomes complex. You can read more about how
+virtual services help with canarying in [Canary Deployments using Istio](/blog/2017/0.1-canary/).
+
+Virtual services also let you:
+
+-   Address multiple application services through a single virtual service. If
+    your mesh uses Kubernetes, for example, you can configure a virtual service
+    to handle all services in a specific namespace. Mapping a single
+    virtual service to multiple "real" services can be particularly useful in
+    facilitating turning a monolithic application into a composite service built
+    out of distinct microservices without requiring the consumers of the service
+    to adapt to the transition: your rules can specify "calls to these URIs of
+    `monolith.com` go to `microservice A`", and so on. You can see how this works
+    in [one of our examples below](#more-about-routing-rules).
+-   Configure traffic rules in combination with
+    [gateways](/docs/concepts/traffic-management/#gateways) to control ingress
+    and egress traffic.
+
+In some cases you also need to configure destination rules to use these
+features, as these are where you specify your service subsets. Specifying
+service subsets and other destination-specific policies in a separate object
+lets you reuse these cleanly between virtual services. You can find out more
+about destination rules in the next section.
+
+### Virtual service example {#virtual-service-example}
+
+Here’s a simple virtual service example. The following virtual service routes
```

Comment:
You should be able to delete the first sentence without impairing the content.

Reply:
done.